src/Symbol.cpp

```diff
@@ -0,0 +1,95 @@
+#include "Symbol.h"
+
+#include <cctype>
+#include <stdexcept>
+#include <functional>
+#include <limits>
+
+namespace calculator {
+    
```

Comment:
// -------------------------------------------------------------------------------------------------------------//
//                                                                                                                                                                //
//                                                                     IInterface Public Methods                                                  //
//                                                                                                                                                                //
// -------------------------------------------------------------------------------------------------------------//

you need to group your functions  visibility order and then by interface. So the same above is for IInterface public methods so you can replace "IInterface" with actual interface value and "Public" with either private, friend, or kepp it public depending on the visibility value